### PR TITLE
FIX: Wildcard domain cannot be given due to AWS tag rule violation

### PR DESCRIPTION
**Pull request by Moondancer83**
Module would be able to handle wildcard domains but AWS tag rules block terraform apply.
I have added a placeholder instead of '*'. Please feel free to suggest a better one or modify it on your own.

### CODE REVIEW

**Review by iuriaranda — APPROVED** [review]

**Comment by iuriaranda** [comment]
Thanks for the contribution. Waiting for another approval from the team and then I'll merge the PR.

**Review by simonrondelez — APPROVED** [review]

**Pull request merged**
